tests/Oryx.Integration.Tests/Php/PhpTwigExampleTest.cs

```diff
@@ -55,5 +55,43 @@ public async Task TwigExample(string phpVersion)
                     Assert.Contains("<h1>Hello World!</h1>", data);
                 });
         }
+
+        [Theory]
+        [InlineData("7.4")]
+        [InlineData("7.3")]
+        [InlineData("7.2")]
```

Comment:
Curious what is the semantic versioning policy in PHP world? Do minor version upgrades have breaking changes as in Python or they are just feature additions?

Reply:
minor version can cause lot of pains (personal experience), php 7.4 is very different than php 7.3.